packages/runtime/test/transport/identityRecoveryKits.test.ts

```diff
@@ -0,0 +1,124 @@
+import { OwnerRestriction } from "../../src";
+import { RuntimeServiceProvider, TestRuntimeServices } from "../lib";
+
+const serviceProvider = new RuntimeServiceProvider();
+
+let services: TestRuntimeServices;
+let servicesWithDisabledDatawallet: TestRuntimeServices;
+
+beforeAll(async () => {
+    const runtimeServices = await serviceProvider.launch(1, { enableDatawallet: true });
+    services = runtimeServices[0];
+
+    const datawalletDisabledServices = await serviceProvider.launch(1, { enableDatawallet: false });
+    servicesWithDisabledDatawallet = datawalletDisabledServices[0];
+}, 30000);
+
+afterAll(async () => await serviceProvider.stop());
+
+afterEach(async () => {
+    const devicesController = services.transport.identityRecoveryKits["createIdentityRecoveryKitUseCase"]["devicesController"];
+    const tokenController = services.transport.identityRecoveryKits["createIdentityRecoveryKitUseCase"]["tokenController"];
```

Comment:
Do the controllers change? Otherwise, this doesn't need to be done again after each test.

Reply:
idk